packages/blade/src/components/VisuallyHidden/VisuallyHidden.d.ts

```diff
@@ -0,0 +1,6 @@
+export { default } from './VisuallyHidden.web';
```

Comment:
Can we have a structure like this?

We can get rid of having to create components twice, having to create `VisuallyHidden.d.ts`, and this matches our component creation pattern

<img width="391" alt="Screenshot 2022-06-15 at 3 37 42 PM" src="https://user-images.githubusercontent.com/24487274/173802387-ede1723d-d6dc-4084-95c7-fcdb60aef62a.png">

Reply:
Attaching more details of each file
<img width="827" alt="Screenshot 2022-06-15 at 3 39 23 PM" src="https://user-images.githubusercontent.com/24487274/173802698-9b512ba8-43ce-4ca9-b42d-bf91d3168c21.png">

Reply:
![Uploading Screenshot 2022-06-15 at 3.39.53 PM.png…]()

Reply:
<img width="729" alt="Screenshot 2022-06-15 at 3 40 07 PM" src="https://user-images.githubusercontent.com/24487274/173802834-12881a6d-96e8-41f5-916e-8aee0627cd83.png">

Reply:
<img width="718" alt="Screenshot 2022-06-15 at 3 40 38 PM" src="https://user-images.githubusercontent.com/24487274/173802970-853ea07f-9477-4e51-af63-f35ab6fbe02e.png">

Reply:
<img width="492" alt="Screenshot 2022-06-15 at 3 40 52 PM" src="https://user-images.githubusercontent.com/24487274/173803031-78484b63-cce7-4a90-895b-13bfbab3acd6.png">

Reply:
That's what I did earlier, changed it after this comment. https://github.com/razorpay/blade/pull/491#discussion_r893313104

cc @kamleshchandnani which one should we use?

Reply:
@chaitanyadeorukhkar understand your point of following pattern consistency but this is just overwhelming hence if you remember we had decided that we'll take this pattern case by case basis as to where the abstraction lies. so i had commented last time to reduce that and have only VisuallyHidden<platform>.tsx

Reply:
Okay, lets keep it this way here then